### PR TITLE
Comment Management (CRUD)

### PR DESCRIPTION
**Pull request by muhammadjon8**

### CODE REVIEW

**Comment by muhammadjon8** [comment]
image
create comment

image
delete comment

image
update comment

image
get all comments

File: src/entities/user.entity.ts [hunk 1]
```diff
@@ -28,6 +29,9 @@ export class User {
   @Column({ default: false })
   isAdmin: boolean
 
+  @OneToMany(() => Comment, (comment) => comment.author)
+  comments: Comment[]
```
Comment (mukhammadjon-ss) [inline src/entities/user.entity.ts:33]: you are saving comments that user write to posts ?

File: src/services/comment.service.ts [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+      where: { blog: { id: blogId } },
+    })
+    return comments
+  }
+
+  async updateComment(blogId: number, commentId: number, content: string) {
+    const comment = await this.commentRepo.findOne({
+      where: {
+        id: commentId,
+        blog: {
```
Comment (mukhammadjon-ss) [inline src/services/comment.service.ts:71]: comments are unique by commentId so I think you don't need to query by blogId

**Review by mukhammadjon-ss — CHANGES_REQUESTED** [review]

File: src/entities/user.entity.ts (hunk 1 above)
Comment (muhammadjon8) [reply src/entities/user.entity.ts:33]: isn't it correct

File: src/services/comment.service.ts (hunk 2 above)
Comment (muhammadjon8) [reply src/services/comment.service.ts:71]: fixed

**Pull request merged**